### PR TITLE
feat(react-native): Xcode plugin includes collect modules script

### PR DESCRIPTION
**Pull request by krystofwoldrich**
context:
- https://github.com/getsentry/sentry-react-native/pull/2606

Maybe this should go to v1.

### CODE REVIEW

**Comment by marandaneto** [comment]
If I got this right, this is based on the current wizard v2, which patches RN >= 0.69 which is the v5 alpha.
https://github.com/getsentry/sentry-react-native/pull/2606 is based on main, which is RN <= 0.68.
So this change has to be either on both wizards v1 and v2, or https://github.com/getsentry/sentry-react-native/pull/2606 has to make it into v5 alpha as well before merging this, and releasing both versions synced with RN releases.

**Comment by krystofwoldrich** [comment]
Added v1 patch https://github.com/getsentry/sentry-wizard/pull/213

**Comment by marandaneto** [comment]
Be sure to test the install and uninstall process directly invoking wizard rather than copy+paste the bash script.
What I'd do is: create an empty project using RN template, run wizard, test it by compiling and guaranteeing that the whole setup works (source map + packages upload on events), uninstall, compile again, etc, if this is tested, LGTM.

**Review by marandaneto — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 - Raise nextjs version limit to include 13 (#206)
 - feat(react-native): Xcode plugin includes collect modules script ([#210](https://github.com/getsentry/sentry-wizard/pull/210))
 
+## 1.4.0
+
+- feat(react-native): Xcode plugin includes collect modules script ([#213](https://github.com/getsentry/sentry-wizard/pull/213))
```
Comment (marandaneto) [inline CHANGELOG.md:10]: I'd only merge after being released tho.

Comment (krystofwoldrich) [reply CHANGELOG.md:10]: Yes, yes.

**Pull request merged**